src/Scss/variables.scss

```diff
@@ -1,4 +1,4 @@
-$application-gray: #C4C4C4;
+$application-gray: rgba( 200, 200, 200, .7);
```

Comment:
we need to make sure the buttons are visible can we add a color #C4C4C4 for the sing in and log out buttons

Reply:
I think chris wrote this.